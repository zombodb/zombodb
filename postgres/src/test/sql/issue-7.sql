SELECT assert((zdb_es_direct_request('idxso_posts', 'GET', '_settings')::json ->
               (zdb_get_index_name('idxso_posts')) -> 'settings' -> 'index' ->> 'number_of_replicas')::bigint, 1,
              'Default replica count of 1');

SELECT assert((zdb_es_direct_request('idxso_users', 'GET', '_settings')::json ->
               (zdb_get_index_name('idxso_users')) -> 'settings' -> 'index' ->> 'number_of_replicas')::bigint, 1,
              'Default replica count of 1');

SELECT assert((zdb_es_direct_request('idxwords', 'GET', '_settings')::json ->
               (zdb_get_index_name('idxwords')) -> 'settings' -> 'index' ->> 'number_of_replicas')::bigint, 1,
              'Default replica count of 1');
