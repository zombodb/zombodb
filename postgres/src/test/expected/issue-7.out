SELECT assert((rest_get(zdb_get_url('idxso_posts') || zdb_get_index_name('idxso_posts') || '.0/_settings') ->
               (zdb_get_index_name('idxso_posts')||'.0') -> 'settings' -> 'index' ->> 'number_of_replicas')::bigint, 1,
              'Default replica count of 1');
 assert 
--------
 t
(1 row)

SELECT assert((rest_get(zdb_get_url('idxso_users') || zdb_get_index_name('idxso_users') || '.0/_settings') ->
               (zdb_get_index_name('idxso_users')||'.0') -> 'settings' -> 'index' ->> 'number_of_replicas')::bigint, 1,
              'Default replica count of 1');
 assert 
--------
 t
(1 row)

SELECT assert((rest_get(zdb_get_url('idxwords') || zdb_get_index_name('idxwords') || '.0/_settings') ->
               (zdb_get_index_name('idxwords')||'.0') -> 'settings' -> 'index' ->> 'number_of_replicas')::bigint, 1,
              'Default replica count of 1');
 assert 
--------
 t
(1 row)

